shared/components/empty-content/empty-content.jsx

```diff
@@ -5,6 +5,11 @@ var React = require( 'react' ),
 	debug = require( 'debug' )( 'calypso:components:emptyContent' ),
 	classNames = require( 'classnames' );
 
+/**
+ * Internal dependencies
+ */
+var FeatureExample = require( 'components/feature-example' );
```

Comment:
hmm, that's how we were doing it until this PR: 
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/jetpack-manage-error-page/index.jsx#L79
but as I was saying at https://github.com/Automattic/wp-calypso/pull/1183#discussion_r46813177, we are using a helper to get all the emptyContent content for every use case in /plugins/, and adding a prop for the feature example were the easier way to add the feature gates there. But looking at the feedback, it seems obvious that it looks odd for everyone... I'll take a look on how to do it without adding the prop.